frontend/src/app/modules/job-status/job-status-modal/job-status.modal.ts

```diff
@@ -151,11 +151,20 @@ export class JobStatusModal extends OpModalComponent implements OnInit {
     }
   }
 
-  private handleDownload(downloadUrl?:string) {
-    if (downloadUrl !== undefined) {
-      this.downloadHref = downloadUrl;
-      // Click download link manually
-      setTimeout(() => this.downloadLink.nativeElement.click(), 50);
+  private handleDownload(redirectionUrl?:string) {
+    if (redirectionUrl !== undefined) {
```

Comment:
I personally like the return-early-and-often pattern, where you do a 

```
if (!redirectionUrl) {
  return;
}

// rest of the code
```

I think it separates conditions nicely from the rest of the code, and reduces nesting